components/eam/src/physics/cam/physpkg.F90

```diff
@@ -2387,7 +2380,7 @@ subroutine tphysbc (ztodt,               &
 
        call stratiform_tend(state, ptend, pbuf, ztodt, &
             cam_in%icefrac, cam_in%landfrac, cam_in%ocnfrac, &
-            landm, cam_in%snowhland, & ! sediment
+            cam_in%snowhland, & ! sediment
```

Comment:
@singhbalwinder , I know you were dealing with an issue in scream that had to do with the land-mask.  Looking at the changes from the upstream merge, is land-mask even going to be used by the ATM model anymore.  It looks like `landm` is getting purged from all the physics processes.